ms365_accountcreator/api/account_creation.py

```diff
@@ -30,13 +31,15 @@ def post(self):
         """
         Create a new account
         """
-        try:
-            LOGIC.create_account(** request.get_json())
-            return "", 204
-        except EmailIllegalError:
-            abort(400, "Email illegal")
-        except EmailAlreadyUsedError:
-            abort(400, "Email already used")
-        except GraphApiError as err:
-            abort(500, "Upstream API error")
-            APP_LOGGER.error(err)
+        lang: str = request.headers.get('lang', 'en')
```

Comment:
Maybe use `request.accept_languages.best_match(['de', 'en'])` to determine the language if the lang header is not set correctly. (See `@BABEL.localeselector` in init)

Reply:
This is not needed when using the same pattern as in routes.py